### ISSUE TITLE
How the pixi is detected

### ISSUE THREAD

**Issue by kresli** [bfanger/pixi-inspector#37]
The real question is how to use it with storybook?
The problem is the storybook content is inside iframe. Exposing pixi to main frame still doesn't work assuming following should work window.PIXI = PIXI; However I believe the extension doesn't look for PIXI variable in main window. So how you detecting the app then?

**Comment by kresli** [comment]
Would be nice if extension would have iframe selector

**Comment by kresli** [comment]
#34

**Comment by kresli** [comment]
I tried to add following
```html
<script>
  window.addEventListener("load", () => {
    const frame = document.getElementById("storybook-preview-iframe");
    const PIXI = frame.contentWindow.PIXI;
    setTimeout(() => {
      window.__PIXI_INSPECTOR_GLOBAL_HOOK__ &&
        window.__PIXI_INSPECTOR_GLOBAL_HOOK__.register({ PIXI: PIXI });
    }, 3000);
  });
</script>
```
but the inspector still looking for pixi.js

**Comment by kresli** [comment]
Also setting everything explicitly in main frame doesn't work. However, this example works in codepen
```html
<body>
  <div id="pixi-canvas"></div>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/5.0.4/pixi.js"></script>
  <script>
    const app = new PIXI.Application({ antialias: true });
    document.getElementById("pixi-canvas").appendChild(app.view);
    console.log(document.getElementById("pixi-canvas"));

    const graphics = new PIXI.Graphics();

    // Rectangle
    graphics.beginFill(0xde3249);
    graphics.drawRect(50, 50, 100, 100);
    graphics.endFill();
    app.stage.addChild(graphics);
  </script>
</body>
```
So the question is if storybook injecting dom elements dynamically and if yes, would be those dynamically elements detected by the extension?

**Comment by kresli** [comment]
Ok, seems like adding
